app/web/src/features/search/FilterDialog.test.tsx

```diff
@@ -158,13 +168,14 @@ describe("FilterDialog", () => {
     expect(numGuestsInput).toHaveValue(3);
 
     userEvent.clear(locationInput);
-    userEvent.clear(lastActiveInput);
+    userEvent.click(lastActiveInput);
+    userEvent.click(screen.getByText("Any"));
```

Comment:
Is this intentional? I noticed that clicking the X in the location input field doesn't clear the filters set in the modal dialog unless we open to clear it there too.

Reply:
Yes - seemed to be the only way to get rid of warnings that "" was not in the list of options. "Any" has a value of null which clears the last active filter. 

Ah yes I have to fix the clearing though - otherwise you can bypass the "filters need location" check

Reply:
Oh yeah, also there's nothing stopping the user from URL hacking the filter right? Does it send a search request to the backend straight away?

Reply:
Yes, but if they encounter weird behaviour that way I think that's on them? They can also set arbitrary location text for a lat/long (not that it has any effect other than cosmetic)